webapp/src/serial.tsx

```diff
@@ -200,7 +200,7 @@ export class Editor extends srceditor.Editor {
         }
         if (this.isVisible) {
             if (this.isCsvView && !this.receivedCsv) {
-                // have not received a csv yet but in csv view; swap back to console.
+                // have not received a csv yet but in csv view; swap back to pxt.
```

Comment:
Not 100% sure what this comment is referring to, but my guess is it should still be "console" here?